src/gwt/src/org/rstudio/core/client/ElementIds.java

```diff
@@ -235,6 +235,7 @@ public static String idFromLabel(String label)
    public final static String MEMORY_PIE_FULL = "memory_pie_full";
    public final static String MEMORY_USAGE_TABLE = "memory_usage_table";
    public final static String MEMORY_TABLE_TITLE = "memory_table_title";
+   public final static String MEMORY_MB_DROPDOWN = "memory_mb_dropdown";
```

Comment:
I think this should just be `memory_dropdown`, because I'm about to remove the `mb` part. https://github.com/rstudio/rstudio/issues/9020

Reply:
The MB here is (I think) a Hungarian-esque "MenuButton" abbreviation. Was copying other example IDs.

Reply:
Though in that case it should be MB_MEMORY_DROPDOWN so I'll change it to that.

Reply:
Went ahead and simplified it to MEMORY_DROPDOWN as suggested.